src/AppInstallerCLICore/Workflows/ImportExportFlow.cpp

```diff
@@ -98,6 +98,7 @@ namespace AppInstaller::CLI::Workflow
     {
         const auto& searchResult = context.Get<Execution::Data::SearchResult>();
         const bool includeVersions = context.Args.Contains(Execution::Args::Type::IncludeVersions);
+        const bool suppressWarnings = context.Args.Contains(Execution::Args::Type::IgnoreWarnings);

```

Comment:
Should this be applied to line 89 as well?

Reply:
Makes sense to me